identityClient/api.go

```diff
@@ -814,3 +817,10 @@ type SearchIdentitiesInformation struct {
 	UserID        string    `json:"user_id"`
 	ClientID      uuid.UUID `json:"client_id"`
 }
+
+// RealmSettingsUpdateRequest wraps the setting available for realm admins to update
+type RealmSettingsUpdateRequest struct {
+	SecretsEnabled      bool     `json:"secrets_enabled"`
+	MFAAvailable        []string `json:"mfa_available"`
+	EmailLookupsEnabled bool     `sql:"email_lookups_enabled"`
```

Comment:
How can the server discriminate between no value provided by the user and the default value that go will give to this struct when deserialized from json?